website/source/docs/agent/options.html.md

```diff
@@ -782,6 +782,9 @@ default will automatically work with some tooling.
       the maximum number of log entries that a server can trail the leader by before being considered unhealthy. Defaults
       to 250.
 
+    * <a name="min_quorum"></a><a href="#min_quorum">`min_quorum`</a> - Sets the minimum number of servers allowed in a cluster
```

Comment:
Allowed doesn't quite fit, maybe again use "needed" or "necessary". Is cluster the right term? Also is there a default? If there isn't a default, then you can say so.

Reply:
This sound better?
```
Sets the minimum number of servers necessary in a cluster
      before autopilot can prune dead servers. There is no default.
```

Reply:
I don't know the proper way to say there isn't a default, but I appreciate the "necessary" wording suggestion. It sounds way more clear then "allowed"